src/routes/routes.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ifAuth from '../components/ifAuth';
+//import LandingPage from '../pages/LandingPage';
+import DiceLanding from '../pages/DiceLanding';
+import Login from '../pages/Login';
+import Signup from '../pages/Signup';
+import { Switch, Route } from 'react-router-dom';
+
+function Routes() {
+    return (
+        <Switch>
+                <Route exact path='/' component = {ifAuth(DiceLanding)} />
+                <Route path='/login' component = {Login} />
+                <Route path='/signup' component = {Signup} />
```

Comment:
Have to check with the team to determine what we want the default page to be when they enter our site. As it stands right now users will automatically be asked to login (or will already be logged in).